### DISCUSSION TITLE
Increase input config back button object bounds size to prevent rulesets from showing under back button

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#17755] [Ideas] [answered]
I really don't know how to title this well so PLEASE rename it for me.
![osu_2022-04-09_23-31-43](https://user-images.githubusercontent.com/26889568/162602981-25486fd0-0d10-4984-a1e5-25d00e575aa9.png)

If you reference the screenshot above you'll see tau appears under the back button when you have a lot of rulesets. IG the object bounds margins need tweaked to account for this or something along those lines.

**Comment by Joehuu** [comment, marked as answer]
Already tracked on one of the points in https://github.com/ppy/osu/issues/13339.
